tests/recipe/CREATE_recipes.test.js

```diff
@@ -0,0 +1,137 @@
+const supertest = require("supertest");
+const app = require("../../app");
+const Recipe = require("../../models/recipe");
+const User = require("../../models/user");
+const api = supertest(app);
+const mongoose = require("mongoose");
+
+const {
+  initialUsers,
+  recipesInDb,
+  recipeToAdd,
+  recipeReview,
+} = require("../test_helper");
+const STATUS_CODE = require("../../shared/errorCode");
+
+const getProperty = async (endpoint, id, property) => {
+  const response = await api.get(`/${endpoint}/${id}`);
+  return response.body[property];
+};
+
+let authHeader;
+let userId;
+let recipeId;
+let token;
+
+describe("recipe api", () => {
+  beforeEach(async () => {
+    await User.deleteMany({});
+    const newUser = initialUsers[1];
+    await api.post("/api/users").send(newUser);
+
+    let response = await api
+      .post("/api/auth/login")
+      .send(newUser)
+      .expect(STATUS_CODE.OK);
+
+    token = response.body.token;
+    authHeader = `Bearer ${token}`;
+    userId = response.body.id;
```

Comment:
You could refector this into a test helper that returns the token so (1) it's easier to understand and (2) I assume you'll probably need to login in other test files.